content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,67 @@
+# Internal link checking tool
+
+You can use [htmltest](https://github.com/wjdp/htmltest) to check for broken links in [`/content/en/`](https://git.k8s.io/website/content/en/). This is useful when refactoring sections of content, moving pages around, or renaming files or page headers.
```

Comment:
Could explain the how the tool works and the limitations (English only?):

[htmltest](https://github.com/wjdp/htmltest) is a tool to check for broken links in HTML output.
Steps to use the tool in this site:
build the HTML content,
pull the `wjdp/htmltest` docker image 
run the tool over the generated HTML content, [`/content/en/`](https://git.k8s.io/website/content/en/). 
Generate a log file (location). Review the log and make corrections?
This is useful when refactoring sections of content, moving pages around, or renaming files or page headers.